.pre-commit-config.yaml

```diff
@@ -0,0 +1,18 @@
+default_language_version:
+  python: python3.6
+
+repos:
+
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.1.0
+    hooks:
+      - id: check-merge-conflict
+      - id: check-yaml
+      - id: check-symlinks
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
+
+  - repo: https://gitlab.com/pycqa/flake8
+    rev: 3.7.7
+    hooks:
+    - id: flake8
```

Comment:
Needs indentation?

Reply:
I think this is actually still valid:

```
In [3]: yaml.safe_load("""
   ...: foo:
   ...: - bar
   ...: """)
Out[3]: {'foo': ['bar']}
```

But agreed it's a bit harder to follow.